includes/plugins/google-site-kit/class-googlesitekitanalytics.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Google Site Kit integration class.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Modules\Analytics_4;
+use Google\Site_Kit\Modules\Analytics_4\Settings;
+use Google\Site_Kit\Core\Modules\Module;
+use Google\Site_Kit\Core\Authentication\Clients\Google_Site_Kit_Client;
+use Google\Site_Kit_Dependencies\Google\Service\GoogleAnalyticsAdmin as Google_Service_GoogleAnalyticsAdmin;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class extending Site Kit's Module, in order to easily access GA data via
+ * Site Kit's Analytics Admin service.
+ */
+class GoogleSiteKitAnalytics extends Module {
```

Comment:
If Site Kit is not installed and active, it throws a fatal error. There should be a proper check before importing this.

Reply:
Done in 4768ceb6bbe31e7f62e8066f4f55b7e75ca292a8